examples/detective/member.yaml

```
@@ -0,0 +1,30 @@
+apiVersion: detective.aws.upbound.io/v1beta1
+kind: Member
+metadata:
+  annotations:
+    upjet.upbound.io/manual-intervention: "Requires AccountId and AccountEmail to replaced manually."
+  labels:
+    testing.upbound.io/example-name: example
+  name: example
+spec:
+  forProvider:
+    accountId: <AccountId>
```

```suggestion
    accountId: ${data.aws_account_id}
```

Let's try to avoid manual intervention.

---

This could not be changed with ${data.aws_account_id}, <AccountId> - it is not current account id, it is any another AWS user we will add as a member. For this test we user free tire user.

---

Done